### ISSUE TITLE
Errata - Critical Rolls/Hits transfer

### ISSUE THREAD

**Issue by HammerGS** [MegaMek/megamek#632]
Their is a rule change regarding Critical Rolls/Hits transfer, we should check to see how MM is handling them.

See here 
http://bg.battletech.com/forums/index.php?topic=7937.0
-and-
http://bg.battletech.com/forums/index.php?topic=23302.0

Relevant Text
_When a location is destroyed, rather than just damaged, no check for critical hits in that location is made unless it contains one or more explosive slots. In that case, any resulting critical hits that strike explosive slots in that location are resolved as normal; hits that do not are always discarded without transferring._

_In other words, you check for crits if you damage a location, but not if you destroy it, unless when it is destroyed it has ammo in it.  Even then, any crits that result never transfer._

**Comment by Dylan-M** [comment]
You also ignore the crits that don't hit ammo, per the last line from the relevant text.

**Comment by rjhancock** [comment]
Closing due to age. Please test with the latest milestone, 0.50.11, to see if the issue still persists. If it does, please re-open.
